ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/RestrictedOffchainEeaSendRawTransaction.java

```diff
@@ -79,8 +78,11 @@ protected Transaction createPrivateMarkerTransaction(
     final String privacyUserId = privacyIdProvider.getPrivacyUserId(user);
 
     final Optional<PrivacyGroup> maybePrivacyGroup =
-        findOffchainPrivacyGroup(
-            privacyController, privateTransaction.getPrivacyGroupId(), privacyUserId);
+        privateTransaction
+            .getPrivacyGroupId()
+            .flatMap(
+                g ->
```

Comment:
I like short names but canonical java does not :shrug:

Reply:
Done